docs/uma-tokenholders/guidence-on-collateral-currency-addition.md

```diff
@@ -0,0 +1,35 @@
+### Guidance for the Addition of Collateral Currencies
+
+Collateral currencies can be added through the collateral currency UMIP template available on Github, by adding 
+ - the name of the token to the be added 
+ - the contract address of the token  together with a link to the address on etherscan
```

Comment:
Does the token have to be ERC20, on Eth chain? It's important to call that out

Reply:
Thanks for highlighting that.  Checking now whether you can have other types on L2 sidechains.